package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@comatch/design-system",
-    "version": "0.24.2",
+    "version": "0.24.0",
```

Favour a version bump instead of a roll-back, even if the last published version is buggy

```suggestion
    "version": "0.24.3",
```

---

will def just patch next time!